symphony-bdk-bom/pom.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>symphony-api-client-java-parent</artifactId>
+        <groupId>com.symphony.platformsolutions</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>symphony-bdk-bom</artifactId>
+    <packaging>pom</packaging>
+    <name>Symphony Java BDK BOM</name>
+    <description>Symphony Java BDK Bom (Bill Of Materials)</description>
+
+    <properties>
+
+        <!-- Compile Dependencies (** Alphabetical order **) -->
+        <apiguardian-api.version>1.1.0</apiguardian-api.version>
+        <bcpkix-jdk15on.version>1.64</bcpkix-jdk15on.version>
+        <commons-io.version>2.6</commons-io.version>
+        <commons-lang3.version>3.9</commons-lang3.version>
+        <lombok.version>1.18.12</lombok.version>
+        <jackson.version>2.11.2</jackson.version>
+        <jackson-databind-nullable.version>0.2.1</jackson-databind-nullable.version>
+        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
```

Comment:
is it normal that we put the plugin's versions in the BOM?

Reply:
Nope :)